__tests__/lib/api/http-client-adapters/fetch-adapter.more-branches.test.ts

```
@@ -6,77 +6,85 @@ global.fetch = mockFetch;
-		const stream = await adapter.stream('/no-body');
-		const iterator = stream[Symbol.asyncIterator]();
-		const first = await iterator.next();
-		expect(first.done).toBe(true);
-		expect(typeof (stream as any).cancel).toBe('function');
-		expect(stream.status).toBe(204);
-	});
+    const stream = await adapter.stream('/no-body');
+    const iterator = stream[Symbol.asyncIterator]();
+    const first = await iterator.next();
+    expect(first.done).toBe(true);
+    expect(typeof (stream as any).cancel).toBe('function');
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)